Library/Homebrew/formula.rb

```diff
@@ -2572,6 +2572,21 @@ def to_hash
     hsh
   end
 
+  def to_bottle_manifest_hashes
+    OnSystem::ALL_OS_ARCH_COMBINATIONS.filter_map do |os, arch|
+      bottle_tag = Utils::Bottles::Tag.new(system: os, arch:)
+      next unless bottle_tag.valid_combination?
+
+      hash = {
+        "name"        => name,
+        "pkg_version" => pkg_version.to_s,
+        "sha256"      => bottle_hash["files"][bottle_tag.to_sym]&.fetch("sha256"),
```

Comment:
Could slim things even more and consider not setting this key at all unless we have a non-nil `sha256` (pulling this into an `if` below)

Reply:
Agreed, makes sense! I figured I might do some testing to see if doing an array instead of a hash is significantly better, too. I wasn't sure whether, once gzipped, that would make a huge difference or not